src/WebAppDIRAC/WebApp/static/DIRAC/PilotMonitor/classes/PilotMonitor.js

```diff
@@ -187,18 +181,6 @@ Ext.define("DIRAC.PilotMonitor.classes.PilotMonitor", {
       computingElement: "Computing Element",
       VO: "VO",
       owner: "Owner",
-      broker: "Broker",
-    };
-
-    var textFields = {
```

Comment:
Seems like `textFields` is still used in the module:

https://github.com/DIRACGrid/WebAppDIRAC/blob/6d6ca9aac0c167b8479f7ca25ef08d4b525bee51/src/WebAppDIRAC/WebApp/static/DIRAC/PilotMonitor/classes/PilotMonitor.js#L197

Also, did you delete `pilotId` on purpose?

Reply:
Good catch! fixed.